doc/search-metrics.org

```diff
@@ -59,6 +59,17 @@ For instance you can search malicious judgments with high confidence matching "a
 GET /ctia/incident/search?query=abuse.ch&confidence=high&limit=10&sort_by=disposition&sort_order=desc
 #+END_SRC
 
+Finally you can only query entity count with the same filtering parameters through the endpoint ~GET /ctia/{entity-type}/search/count~.
```

Comment:
same remark as I do most of the time.
I think using GET parameters while better to respect HTTP protocol is
harder and less composable than using a full JSON body as a request.
This is why I think you should propose you should duplicate the handler with another
POST route that uses a JSON body instead of query parameters.

Reply:
I think it worth doing that but I just documented existing route.